config/dosarray_config.sh

```diff
@@ -74,12 +74,16 @@ export DOSARRAY_CONTAINER_HOST_IDXS=`dosarray_physical_hosts_skip`
 export DOSARRAY_VIRT_INSTANCES=10
 export DOSARRAY_MIN_VIP=2
 export DOSARRAY_MAX_VIP=$((DOSARRAY_VIRT_INSTANCES + (DOSARRAY_MIN_VIP - 1)))
-
-# -t is required to execute network configuration commands with sudo
 function dosarray_execute_on () {
   local HOST_NAME="$1"
   local CMD="$2"
-  ssh <USERNAME>@${HOST_NAME}.<FULLY_QUALIFIED_NAME> -p <SSH_PORT> -t ${CMD}
+  local SSH_PARAM="$3"
+  if [ -z ${SSH_PARAM} ]
+  then
+    ssh <USERNAME>@${HOST_NAME}.<FULLY_QUALIFIED_NAME> -p <SSH_PORT> ${CMD}
+  else
+    ssh <USERNAME>@${HOST_NAME}.<FULLY_QUALIFIED_NAME> -p <SSH_PORT> -t ${CMD}
```

Comment:
For generality this line could be put as
`ssh <USERNAME>@${HOST_NAME}.<FULLY_QUALIFIED_NAME> -p <SSH_PORT> ${SSH_PARAM} ${CMD}`
since down the line `SSH_PARAM` might need to be a different/additional parameter to `-t`.

Additionally, the `if [ -z ${SSH_PARAM} ]` test wouldn't be necessary, since both cases would map to the line above (since `${SSH_PARAM}`="" in the first case).